packages/data-portal-explore/src/components/FileFilterControls.tsx

```diff
@@ -151,6 +152,31 @@ export const FileFilterControls: React.FunctionComponent<IFileFilterControlProps
                             });
                     }}
                 />
+                <FilterDropdown
+                    {...dropdownProps}
+                    attributes={[AttributeNames.viewers]}
+                    options={(attrName: AttributeNames) => {
+                        return options(attrName).map((e: OptionType) => {
+                            const viewerLabels = {
+                                [FileViewerName.minerva]: 'Minerva',
+                                [FileViewerName.ucscXena]: 'USCS Xena',
+                                [FileViewerName.cellxgene]: 'CellxGene',
+                                [FileViewerName.isbcgc]: 'BigQuery',
+
+                                // excluded or aggregated values:
+                                // we are not supposed to see these as filter options
+                                [FileViewerName.autoMinerva]: 'Auto Minerva', // aggregated as Minerva
+                                [FileViewerName.customMinerva]:
+                                    'Custom Minerva', // aggregated as Minerva
```

Comment:
We have an option to have both these filters (`Auto Minerva` and `Custom Minerva`) in the filter menu. We don't have to aggregate them as `Minerva`.